src/components/course/CoursePage.jsx

```diff
@@ -1,12 +1,110 @@
-import React from 'react';
```

Comment:
This file is copy/pasted from `Course.jsx` which since deleted. The previous `CoursePage` was only rendered `Course` so these 2 components could be consolidated.

Same with the `Dashboard` and `DashboardPage` component files below.